x-pack/plugins/monitoring/server/telemetry_collection/get_ingest_solutions.ts

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { SearchResponse } from 'elasticsearch';
+import { StatsCollectionConfig } from 'src/plugins/telemetry_collection_manager/server';
+// @ts-ignore
+import { buildGetIndicesQuery } from '../lib/elasticsearch/indices';
+import { INDEX_PATTERN_ELASTICSEARCH } from '../../common/constants';
+import {
+  buildIngestSolutionsPayload,
+  IngestSolutionsIndex,
+} from '../../../../../src/plugins/telemetry/server/';
+
+interface IndexStatsSearchResponse {
+  index_stats: {
+    index: string;
+    primaries: {
+      docs: {
+        count: number;
+      };
+    };
+    total: {
+      store: {
+        size_in_bytes: number;
+      };
+    };
+  };
+}
+
+export function handleResponse(response: SearchResponse<IndexStatsSearchResponse>) {
+  const reducedIndices: IngestSolutionsIndex[] = (response.hits?.hits || []).map(({ _source }) => ({
+    name: _source.index_stats.index,
+    docCount: _source.index_stats.primaries.docs.count,
+    sizeInBytes: _source.index_stats.total.store.size_in_bytes,
+  }));
+
+  return buildIngestSolutionsPayload(reducedIndices);
+}
+
+export async function getIngestSolutions(
+  callCluster: StatsCollectionConfig['callCluster'],
+  clusterUuids: string[],
+  start: StatsCollectionConfig['start'],
+  end: StatsCollectionConfig['end'],
+  maxBucketSize: number
+) {
+  const responses = await Promise.all(
+    clusterUuids.map(async clusterUuid => {
+      // Should we take into consideration CCS? https://github.com/elastic/kibana/blob/3a396027f669803e1a3143237578973fb1ab20d0/x-pack/plugins/monitoring/server/routes/api/v1/elasticsearch/indices.js#L42
```

Comment:
That's a good question. Are there any repercussions if we do use CCS by default? (like will it be less efficient, or slower?). If not then my other concern is it would probably need to be conditional based on licensing (and maybe also if there are any config options tied with it). I think the terms are kind of confusing since we actually mean "Multi-stack monitoring" here, right?
![Screen Shot 2020-05-05 at 11 36 10 AM](https://user-images.githubusercontent.com/27822304/81085354-1f1e4380-8ec5-11ea-940e-cc2a57b53e2f.png)

Looks like it's only available for Gold license

---

However, I think CSS is available for all licenses:
![Screen Shot 2020-05-05 at 11 36 58 AM](https://user-images.githubusercontent.com/27822304/81085507-4d9c1e80-8ec5-11ea-82ac-c7accf602141.png)

Source: https://www.elastic.co/subscriptions

I think it's fine the way it is right now and can be added later if anything